Snail/Observable.swift

```diff
@@ -297,33 +297,40 @@ public class Observable<T>: ObservableType {
     public static func combineLatest<U>(_ input1: Observable<T>, _ input2: Observable<U>) -> Observable<(T, U)> {
```

Comment:
maybe in the future you want to make a variadic version of this! :)

Reply:
I wish! Swift doesnt offer variadic generics though ):

Reply:
Didn't know! good stuff!